.github/workflows/daily-nightly-jobs.yml

```diff
@@ -26,20 +26,27 @@ jobs:
         with:
           go-version: 1.18
 
-      - name: teardown minikube and docker
+      - name: teardown minikube, docker and kubectl
+        if: always()
```

Comment:
Actually, if these cleanup actions are at the end with the `if: always()`, wouldn't that be a good enough guarantee that it's cleaned up for the next run? Then the minikube won't be left running while there is no job running.

Reply:
yes, changed